.github/actions/free-disk-space/action.yml

```diff
@@ -5,5 +5,9 @@ runs:
   steps:
     - shell: bash
       run: |
-        # Run in background to save time
-        sudo rm -rf /usr/local/lib/android &
+        df -h
+        sudo rm -rf /usr/local/lib/android
+        sudo rm -rf /usr/share/dotnet
+        sudo rm -rf /opt/ghc
+        sudo rm -rf /usr/local/share/boost
+        df -h
```

Comment:
This adds 4~5 minutes overhead to each job. Filed https://github.com/mlflow/mlflow/pull/18154 as a quick term solution.

Reply:
Sounds good - will close this PR